PProPanel/jsx/general.jsx

```diff
@@ -0,0 +1,6 @@
+if (typeof $ == 'undefined') {
```

Comment:
>Well it doesn't hurt to do the extra check, if there's already a $ object defined then it won't replace it.

I agree with the exception that it _suggests_ that the `$` object is similar to that in pure JavaScript - something libraries tend to use. In ExtendScript it is something special and always included by default unless you did something strange. It raises questions.

Likewise, though, I would get @bbb999's thoughts before making such a change :D